--- src/potential/electrostatics/helpme.h.bak	2026-02-21 12:42:47.029674719 -0500
+++ src/potential/electrostatics/helpme.h	2026-02-21 12:43:27.487790201 -0500
@@ -3707,7 +3707,11 @@
             rPower_ = rPower;
             algorithmType_ = algorithm;
             splineOrder_ = splineOrder;
+#if defined(__MINGW64__)
+            cacheLineSizeInReals_ = static_cast<Real>(4096 / sizeof(Real));
+#else
             cacheLineSizeInReals_ = static_cast<Real>(sysconf(_SC_PAGESIZE) / sizeof(Real));
+#endif
             requestedNumberOfThreads_ = nThreads;
 #ifdef _OPENMP
             nThreads_ = nThreads ? nThreads : omp_get_max_threads();
--- plugins/Makefile.in.bak	2026-02-21 12:53:30.903657498 -0500
+++ plugins/Makefile.in	2026-02-03 17:54:02.000000000 -0500
@@ -306,7 +306,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = i-pi
+SUBDIRS = # i-pi
 ACLOCAL_AMFLAGS = -I m4
 all: all-recursive
 
--- configure.bak	2026-02-21 15:24:57.672889417 -0500
+++ configure	2026-02-21 15:27:44.051424341 -0500
@@ -18108,7 +18108,10 @@
 
 # Checks for library functions.
 
-
+if echo ${CXX} | grep w64-mingw > /dev/null
+then
+    CPPFLAGS="${CPPFLAGS} -DM_PI=3.14159265358979323846"
+fi
 
 cat >confcache <<\_ACEOF
 # This file is a shell script that caches the results of configure
